.github/workflows/instant_benchmark.yml

```diff
@@ -67,6 +67,17 @@ jobs:
       - name: Clean env
         run: |
           yes | docker system prune -a --volumes
+      - name: install deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install awscli -y
```

Comment:
use pip install awscli

Reply:
Right now, we use the apt-get install in most places in CI with only a single exception using the pip install. It would be better to follow the convention in this PR and then change all usages to `pip install` in a following one